tests/edit_inputs.sh

```diff
@@ -55,3 +56,46 @@ function edit_diag_table {
       -e "s/SMONTH/$SMONTH/g" \
       -e "s/SDAY/$SDAY/g"
 }
+
+function edit_ww3_input { 
+
+  SDATEWW3="${SYEAR}${SMONTH}${SDAY} $(printf "%02d" $(( ${SHOUR}  )))0000"
+  EDATEWW3="2100${SMONTH}${SDAY} $(printf "%02d" $(( ${SHOUR}  )))0000"
+  #note EDATE should be SDATE+FHMAX, but since this requires adding ndate 
+  #a work around is to just put a date long in the future as the actual end time is
+  #determined by the driver
+  DT_2_RST_WAV="$(printf "%02d" $(( ${WW3RSTDTHR}*3600 )))"
+  DTFLD_WAV="$(printf "%02d" $(( ${WW3OUTDTHR}*3600 )))"
+  DTPNT_WAV="$(printf "%02d" $(( ${WW3OUTDTHR}*3600 )))"
+
+  sed -e "s/NFGRIDS/$NFGRIDS/g" \
```

Comment:
Multiple `-e` options doesn't work in POSIX sed. Also, why don't you use atparse instead? That's what it's for.